includes/Helpers/BanHelper.php

```diff
@@ -19,8 +19,7 @@ class BanHelper implements IBanHelper
 	 */
 	private $database;
 
-	public function __construct(PdoDatabase $database)
-	{
+	public function __construct(PdoDatabase $database) {
```

Comment:
You may want to check your code style settings here - I'm fairly sure we should be using brace on new line for methods/classes